docs/contributing/formatting.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Code Formatting Guidelines"
+summary: "PCSX2's Formatting Guidelines"
+draft: false
+toc: true
+sidebar_position: 2
+---
+
+Basic code formatting guide that you should take into account.
+
+To help with code readability, please follow these basic code‐formatting guidelines. Doing so will greatly help both you, other contributors, the PCSX2 team and everyone else.
+
+## Topic 1: Do not use variable type prefixes
+
+Variable type prefixes are commonly known as [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation), and are the sort of variable names popularized by the `Win32API`. They look something like this is in ideal situations:
+
+```C++
+DWORD dwSomeDoubleWord;
+```
+
+Besides being unreadable and looks terrible, this naming convention is totally impractical for modern programming via an IDE with autocomplete features. Coders using such tools want to do autocomplete by the subject of the variable's purpose, and are then automatically provisioned variable type information via tooltip, making the prefix neigh worthless. If you're looking for the `MasterVolume` of a voice, you might think `VoiceVolumeMaster`, `MasterVolumeVoice`, or `VolumeVoiceMaster` and in fact you'll probably have no idea of it's a dword, word, int, or float value type. Any code using this naming convention will be changed to something more sensible.
```

Comment:
Guessing "neigh" is meant to be "nigh"? May be better to use a simpler word like "almost" or "nearly".